project2/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Rubik:400,500&display=swap" rel="stylesheet">
+    <title>Movie Card</title>
+</head>
+
+<body>
+    <div>
+        <!--image was found on Google-->
+        <img src="images/YTuMamaTambien.jpg">
+        <section class="info">
+            <h2>Y Tu Mamá También</h2>
+            <div class="stars">
+                <svg width="24" height="24" viewBox="0 0 24 24">
+                    <path
+                        d="M12,17.27L18.18,21L16.54,13.97L22,9.24L14.81,8.62L12,2L9.19,8.62L2,9.24L7.45,13.97L5.82,21L12,17.27Z">
+                    </path>
+                </svg>
+                <svg width="24" height="24" viewBox="0 0 24 24">
+                    <path
+                        d="M12,17.27L18.18,21L16.54,13.97L22,9.24L14.81,8.62L12,2L9.19,8.62L2,9.24L7.45,13.97L5.82,21L12,17.27Z">
+                    </path>
+                </svg>
+                <svg width="24" height="24" viewBox="0 0 24 24">
+                    <path
+                        d="M12,17.27L18.18,21L16.54,13.97L22,9.24L14.81,8.62L12,2L9.19,8.62L2,9.24L7.45,13.97L5.82,21L12,17.27Z">
+                    </path>
+                </svg>
+                <svg width="24" height="24" viewBox="0 0 24 24">
+                    <path
+                        d="M12,17.27L18.18,21L16.54,13.97L22,9.24L14.81,8.62L12,2L9.19,8.62L2,9.24L7.45,13.97L5.82,21L12,17.27Z">
+                    </path>
+                </svg>
+                <svg width="24" height="24" viewBox="0 0 24 24">
+                    <path
+                        d="M12,17.27L18.18,21L16.54,13.97L22,9.24L14.81,8.62L12,2L9.19,8.62L2,9.24L7.45,13.97L5.82,21L12,17.27Z">
+                    </path>
+                </svg>
+            </div>
+            <div>
+                <a class="play" href="https://www.youtube.com/watch?v=UW1LQbtRuAM">
```

Comment:
Consider that you have a class of "play" and one of "playbutton" -- it would be pretty easy for someone else to put the wrong styles on the wrong class because they're not declarative enough to know which is which without looking at the markup. Something like "play-link" and "play-icon" would make them more distinguishable.